### PR TITLE
Feature/fov export

### PR DESCRIPTION
**Pull request by dabeshouse**
This is corrects the export of Camera fov and clipping plane export. It is in the IDE. It is in the tweedle library. It is in the player.
This is mainly to correct export. It should have been merged a few months back when the other changes went in

### CODE REVIEW

File: core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java [hunk 1]
```diff
@@ -54,4 +55,22 @@
 public class SymmetricPerspectiveCamera extends AbstractPerspectiveCamera {
   public final AngleProperty verticalViewingAngle = new AngleProperty(this, new AngleInRadians(0.5), true);
   public final AngleProperty horizontalViewingAngle = new AngleProperty(this, new AngleInRadians(Double.NaN), true);
+  private AngleInRadians effectiveHorizontalViewingAngle;
```
Comment (jennej) [inline core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java]: this feels nitpicky, but I feel like the getters and setters should use the same type?

Comment (jennej) [reply core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java]: and then i looked into the difference between AngleInRadians and Angle and i think i sprained something.

**Review by jennej — APPROVED** [review]

File: core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java (hunk 1 above)
Comment (dabeshouse) [reply core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java]: Not so picky. I have a mix of revolutions and radians along with wrapped and unwrapped values that are not always clear from the method or variable name.
I will see if I can make this a little clearer.

Comment (dabeshouse) [reply core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/SymmetricPerspectiveCamera.java]: I made it only a little clearer.

**Pull request merged**